apps/inbound-mail/src/.env.development

```diff
@@ -7,3 +7,8 @@ HOST="0.0.0.0"
 REDIS_DB_INDEX="2"
 REDIS_HOST="localhost"
 REDIS_PORT="6379"
+
+NEW_RELIC_ENABLED=true
+NEW_RELIC_APP_NAME="[DEV] - INBOUND-MAIL"
```

Comment:
Can we make those variable to be passed from terraform? so we can have consistent naming for Prod environments (US/EU and etc...)

Reply:
Absolutely

Reply:
Done

Reply:
I guess we can remove this from here if that's the case :)